manticore/utils/event.py

```diff
@@ -151,3 +156,7 @@ def forward_events_to(self, sink, include_source=False):
         if not isinstance(sink, Eventful):
             raise TypeError
         self._forwards[sink] = include_source
+
+    def copy_eventful_state(self, new_object: 'Eventful'):
```

Comment:
```suggestion
    def copy_eventful_state(self, new_object: Eventful):
```

isn't it working this way?

Reply:
nope, Eventful is not declared yet